### ISSUE TITLE
Blocked at: What do you want to do about modified configuration file php.ini?

### ISSUE THREAD

**Issue by idflood** [laravel/homestead#577]
## Versions
  - Vagrant: 1.9.5
  - Provider: Virtualbox
  - Homestead: 5.3.1 (box 2.1.0)

## Host operating system
macOS Sierra 10.12.4

## Homestead.yaml
```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/Sites
      to: /home/vagrant/Sites
      type: "nfs"

sites:
    - map: test.dev
      to: /home/vagrant/Sites/test/web
      type: apache

databases:
    - homestead
```

## Vagrant up output
https://gist.github.com/idflood/e63898b0cbc16adf4287837730bb6178

## Expected behavior
Vagrant up should finish

## Actual behavior
Blocked at a dialog asking what to do with php.ini file.

## Steps to reproduce
1. Followed the steps here https://laravel.com/docs/5.4/homestead#first-steps
2. Did `git checkout v5.3.1`
3. Modified the Homestead.yaml as shown above
4. vagrant up

The vagrant output is a bit messy I think, the "real last line" should be:
`==> homestead-7: Package configuration┌──────────────────────┤ Modified configuration file ├──────────────────────┐│ A new version (/usr/lib/php/7.1/php.ini-production.cli) of configuration  ││ file /etc/php/7.1/cli/php.ini is available, but the version installed││ currently has been locally modified.││││ What do you want to do about modified configuration file php.ini?││││install the package maintainer's version││keep the local version currently installed         ││show the differences between the versions││show a side-by-side difference between the versions││start a new shell to examine the situation││││││<Ok>│││└───────────────────────────────────────────────────────────────────────────┘`

Eventually I did not modify this file and there seems to be no way of answering this dialog.

**Comment by svpernova09** [comment]
[5.3.2](https://github.com/laravel/homestead/releases/tag/v5.3.2) Should fix this. Let me know if it doesn't. (Works on my machine™) :D

**Comment by idflood** [comment]
Thanks @svpernova09 this is working fine now 👍

**Comment by dimitriadisg** [comment]
Awesome !
I can't believe this fix came just 11 days ago, I'm lucky, thank you !
Confirming that fix is working for Windows 10.

**Comment by jonnywilliamson** [comment]
I'm still getting this.

Just after checking out tag v5.3.2 

Vagrant destroy
Vagrant up

![image](https://user-images.githubusercontent.com/2513663/26930033-ebe158c4-4c53-11e7-90f0-8fbee386377a.png)


Anything else I should be doing?

**Comment by jonnywilliamson** [comment]
Currently running version 5.3.2 on box 2.1.0
![image](https://user-images.githubusercontent.com/2513663/26930354-10458a86-4c55-11e7-8d7c-6e32991d172b.png)

**Comment by svpernova09** [comment]
@jonnywilliamson please open a new issue and include all the requested files, also include your `after.sh` file

**Comment by jonnywilliamson** [comment]
Just discovered its due to a 

`apt-get update -y` in the first line of my `after.sh` file. Thanks for the hint. I'll have to remove that and do it manually once the machine is ready.
